boards/arc/em_starterkit/em_starterkit.dts

```diff
@@ -14,6 +14,20 @@
 	compatible = "snps,em_starterkit-em9d", "snps,em_starterkit";
 
 	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		led2 = &led2;
+		led3 = &led3;
+		led4 = &led4;
+		led5 = &led5;
+		led6 = &led6;
+		led7 = &led7;
+		led8 = &led8;
+		sw0 = &button0;
+		sw1 = &button1;
+		sw2 = &button2;
+		uart-0 = &uart0;
```

Comment:
copy-paste error: 3 times uart-0?
The em_starterkit, em_starterkit_em7d, em_starterkit_em11d all share the same baseboard, possible to factor out the leds & button definition and move to the common emsk.dtsi?

Reply:
Fixed copy paste error, refactor common bits into board.dtsi that is now shared.